docs/platforms/react-native/manual-setup/manual-setup.mdx

```diff
@@ -111,14 +111,6 @@ export SENTRY_CLI_DEBUG_FILES_UPLOAD_EXTRA_ARGS="--extra --flags"
 
 For more information about what options to use in `SENTRY_CLI_DEBUG_FILES_UPLOAD_EXTRA_ARGS` visit the [Sentry CLI Apple Debug Symbols](/platforms/apple/dsym/#sentry-cli) documentation.
 
-### Linking the Native Library via CocoaPods
-
-To link the native library via CocoaPods, link to the native module in your Podfile. In React Native versions 0.60 and above, the podspec should be automatically linked. If isn't, you can link to the native module manually by adding the `RNSentry.podspec` to your Podfile as follows:
-
-```ruby
-pod 'RNSentry', :path => '../node_modules/@sentry/react-native/RNSentry.podspec'
-```
-
 ### Using node with nvm or Volta
 
 If you are using nvm or Volta, Xcode has trouble locating the default node binary, which can look similar to the [troubleshooting example](/platforms/react-native/troubleshooting/#using-node-with-nvm-or-volta).
```

Comment:
```suggestion
If you're using nvm or Volta, Xcode may have trouble locating the default node binary. The suggested workaround in this [troubleshooting example](/platforms/react-native/troubleshooting/#using-node-with-nvm-or-volta) can help.
```
Since we're removing the "how to" portion of this section, it helps to be more direct in sending them to troubleshooting docs :)